src/component/Header/HeaderResponsive.scss

```diff
@@ -0,0 +1,18 @@
+//For Tabs
+@media (min-width:768px) and (max-width:990px){}
+
+//For Mobile Landscape
+@media (max-width:767px){
+  #main-header{
+    h1 {font-size: 20px;}
+  }
+}
+
+//For Mobiles
+@media (max-width:600px){}
+
+//For small Mobiles
+@media (max-width:320px){}
+
+//For Ipad Pro and mini laptops
+@media screen and (min-width:991px) and (max-width:1200px){}
```

Comment:
@mdameerhamza Didn't we agree on single file containing the media queries ?
@MAslanweb We have to move these width into SCSS variables.